src/RestControllers/TransactionRestController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * PatientRestController
```

Comment:
Please provide at least a one sentence description of what this class does so our document generators will have it.